Adyen.EcommLibrary.Test/BaseTest.cs

```diff
@@ -1,22 +1,23 @@
 ﻿using Adyen.EcommLibrary.Constants;
-using Adyen.EcommLibrary.HttpClient;
-using Adyen.EcommLibrary.HttpClient.Interfaces;
-using Adyen.EcommLibrary.Model;
+using Adyen.EcommLibrary.HttpClientHandler;
+using Adyen.EcommLibrary.HttpClientHandler.Interfaces;
 using Adyen.EcommLibrary.Model.Modification;
+using Adyen.EcommLibrary.Model.Nexo;
 using Adyen.EcommLibrary.Service;
 using Moq;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+using Environment = System.Environment;
 
 namespace Adyen.EcommLibrary.Test
 {
     public class BaseTest
     {
 
         #region Payment request
-        public PaymentResult CreatePaymentResultFromFile(string fileName)
+        public Model.PaymentResult CreatePaymentResultFromFile(string fileName)

```

Comment:
Why Model.PaymentResult can we not do PaymentResult ?

Reply:
The reason is now there are two PaymentResult classes. One generated from nexo standards and one generated from swagger for ecom. It can make more clear by adding 
using PaymentResult = Adyen.EcommLibrary.Model.PaymentResult;